### DISCUSSION TITLE
My controller

### DISCUSSION THREAD

**Discussion by willmodelisme** [shanteacontrols/OpenDeck#245] [Show and tell]
Just finished a new midi controller with opendeck.
8 faders, 8 potentiometers, 10 rotary/switch encoder and a IC2 OLED display.
3D printed shell.

![y7AnWciaStuFIVocaa0foQ](https://github.com/shanteacontrols/OpenDeck/assets/122353083/ebf7e4d2-2f62-4539-91dd-9f8339817dca)
![0c07sviZTwimbPhRL84YHA](https://github.com/shanteacontrols/OpenDeck/assets/122353083/d4129d09-9bd5-47b5-bba2-91badd707968)

**Comment by paradajz** [comment]
Looks absolutely amazing. The orange is stunning.

**Comment by paradajz** [comment]
Out of curiousity: which encoders are you using?

**Comment by willmodelisme** [comment]
Encoder are from Amazon
https://www.amazon.fr/gp/product/B09SG3HF9N/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1

**Reply by Babylonrising** [reply]
Hey! Are you getting the encoders to work smoothly with having to turn them 5 times?
